packages/lit-dev-content/site/tutorials/content/first-component/06.md

```diff
@@ -0,0 +1,140 @@
+In this section we'll make it so when you click on the `word-viewer` the
+animation plays in reverse.
+
+Before adding the event handling we need a new piece of internal state to
```

Comment:
you haven't introduced event handling. Perhaps just ignore that and start with

> First, create the `playDirection` internal state property which will represent...

Reply:
oops! Great catch!